.travis.yml

```diff
@@ -23,17 +25,17 @@ script:
 
 after_success:
   - codecov
+  - if [[ $BUILD_INSTALLER == 'yes' ]]; then pip install pynsist; ./script/makeinstaller.sh; fi
```

Comment:
Sorry for nitpicking here, but the `after_success` life cycle hook ignores failures:
https://docs.travis-ci.com/user/customizing-the-build/#Breaking-the-Build

> The exit code of after_success, after_failure, after_script and subsequent stages do not affect the build result. However, if one of these stages times out, the build is marked as a failure.

What I meant was adding the ./script/makeinstaller.sh call to the `script` hook.

Regarding the pip dependencies:
Wouldn't it make much more sense adding this to `before_install` hook, since there's already a list of pip dependencies?